### ISSUE TITLE
Available data

### ISSUE THREAD

**Issue by markacolville** [jordanchubb/superprice#1]
Any chance that the full dataset of products can be made available if you still have it? It's very hard to find good time-series data of supermarket prices going back.

Cheers

**Comment by jordanchubb** [comment]
Yes I still have the data.  I don't have the time at the moment to make it available sorry.
Let me know if you have big plans for the data and I could get the raw data for you.

The data I have is only over a 6 month period - up until my access was restricted.

**Comment by markacolville** [comment]
No big plans sadly until I can find a longer stretch of data to dig into. I want to look into shrinkflation, sneaky product changes etc, but getting good stretch of data is hard but anything is better than nothing.

No rush, but even just raw csv/jsons/html pushed to a side branch would be amazingly useful.
